scripts/nodelib.py

```diff
@@ -17,11 +18,22 @@ def install_cargo():
         subprocess.call([os.path.expanduser('~/.cargo/bin/cargo'), '--version'])
     except OSError:
         print("Installing Rust...")
-        subprocess.check_output('curl https://sh.rustup.rs -sSf | sh -s -- -y', shell=True)
+        subprocess.check_output('curl https://sh.rustup.rs -sSf | sh -s -- -y --default-toolchain nightly-2019-10-04', shell=True)
```

Comment:
why do you need to specify toolchain? I thought cargo will pick up the toolchain file automatically